guides/hack/00-overview/03-async.md

```diff
@@ -1,85 +1,15 @@
 ### Async Programming in Hack

```

Comment:
Why are we starting with `###` instead of `#` or `##` at the very least?

I was careful to try to not to get to 4-levels of headings in any of the content.